bin/interpret/funcfile.py

```diff
@@ -92,24 +92,42 @@ def FUnC(dataDict, binDict, cutoffDict, gender):
 			
 	return dataDict
 
-
-
+####original code, but seems to just compare CNVs to themselves?
```

Comment:
So, this shouldn't be comparing CNVs to themselves, unless I've forgotten something about how python works (always possible). 

Let's say you have funcDict ={A, B, C} and then start going through the loop  for i,j in enumerate(funcDict[1:]). On the first round, i == 0 and j == B. So testing j['pass'] == funcDict[i]['pass'] should be equivalent to B['pass'] == A['pass'].

The changes you've made look totally fine. But there will be cases where they aren't comparing the same thing--when there are > 2 CNV segments with matching copy numbers in a row. That shouldn't happen, but it's something that should be double checked on some actual data...just in case that makes some part of this function freak out. 

Otherwise I'm fine with using either version and having the other one commented out (maybe with a header comment about being an alternate loop structure).

Reply:
! This is the change I was the most confused/hesitant about because I had tried to replicate it in just in python to try to follow the logic better and thought it would be peculiar if you hadn't noticed that it was comparing CNVs to themselves... I double checked it again and it does work as you had it originally (I'm now not sure what I did earlier when it didn't seem to work), so might as well keep it that way.

Reply:
Sure, I'm fine with keeping it the same as the original version...though since you did see some kind of failure previously it's probably a good idea to keep a backup strategy commented out in there. Unless I've overthinking things :P

Reply:
I'll do some tests with stuff commented out before I straight up delete things, and we'll see.